File: assistant/ai-assistant-api-old-python/models/post.py
from typing import Union, List, Optional

from pydantic import BaseModel

class Post(BaseModel):
    post_id: Optional[int] = None
    name: str
    description: str
    language: str
    group_name: str
    image_url: Optional[str] = None
    long_lat: Optional[str] = None
    counter_endorsements_up: int
    counter_endorsements_down: int
    counter_points_for: Optional[int] = 0
    counter_points_against: Optional[int] = 0
    points_for: str
    points_against: str
    total_number_of_posts: Optional[int] = 0
    group_id: int
    community_id: int
    domain_id: int
    cluster_id: Optional[int] = None
    date: str
    status: str
    official_status: Optional[str] = None

File: server_api/ts-out/active-citizen/engine/assistant/ai-assistant-api-unused/models/post.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const typescript_optional_1 = require("typescript-optional");
const pydantic_1 = require("pydantic");
class Post extends pydantic_1.BaseModel {
    post_id = typescript_optional_1.Optional.empty();
    name;
    description;
    language;
    group_name;
    image_url = typescript_optional_1.Optional.empty();
    long_lat = typescript_optional_1.Optional.empty();
    counter_endorsements_up;
    counter_endorsements_down;
    counter_points_for = typescript_optional_1.Optional.of(0);
    counter_points_against = typescript_optional_1.Optional.of(0);
    points_for;
    points_against;
    total_number_of_posts = typescript_optional_1.Optional.of(0);
    group_id;
    community_id;
    domain_id;
    cluster_id = typescript_optional_1.Optional.empty();
    date;
    status;
    official_status = typescript_optional_1.Optional.empty();
}
